assets/javascripts/discourse/components/chat-channel-delete-modal-inner.js

```diff
@@ -0,0 +1,55 @@
+import Component from "@ember/component";
+import { isTesting } from "discourse-common/config/environment";
+import { later } from "@ember/runloop";
+import { isEmpty } from "@ember/utils";
+import I18n from "I18n";
+import discourseComputed from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import { ajax } from "discourse/lib/ajax";
+import { inject as service } from "@ember/service";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Component.extend({
```

Comment:
I think it is worth having an acceptance test here to test the whole deletion flow.

Reply:
Ah yep no problem, will add one, cause I have one for Archive + Change Status too